src/leo-server-app/src/server.ts

```diff
@@ -1,9 +1,14 @@
 import { scheduleJobsForSatellitesOnBoot } from "./utils/satellite.utils";
 
 const { connectDB } = require("./database/database");
+const SocketServer = require("./socket");
 const app = require("./app");
 const port = process.env.PORT || 8080;
 
+SocketServer.listen(1549, () => {
+  console.log("Listening on port: 1549");
+});
```

Comment:
what is port 1549?

Reply:
The port that is listening for socket connections

Reply:
this port serves https connections (socket) and the other port services http requests (api reqs)

Reply:
is it always going to be this port? If not prob better to have a constant for it and have the varialbe be used for the console.log

Reply:
yeah true, added constant